app/code/Magento/InventoryInStorePickupApi/Model/GetPickupLocationInterface.php

```diff
@@ -4,13 +4,13 @@
  * See COPYING.txt for license details.
  */
 
-namespace Magento\InventoryInStorePickupApi\Api;
+namespace Magento\InventoryInStorePickupApi\Model;
```

Comment:
Why?

Reply:
This is nice sugar-service.
As API (inc webapi) we have `GetPickupLocations` service.